app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
```

Comment:
10 seems too short; I think I heard @clu say '3 lines' at one point? Maybe an idea of a desired number of words would be more helpful; we could then use the avg English word length or something.